openscm/__init__.py

```diff
@@ -8,6 +8,7 @@
     convert_scmdataframe_to_core,
     convert_core_to_scmdataframe,
     convert_config_dict_to_parameter_set,
+    df_append,
 )
 from .adapters import get_adapter
 from .constants import ONE_YEAR_IN_S_INTEGER
```

Comment:
@znicholls I thought you don't want to use fixed time intervals?

Reply:
I don't, this is a hack to see if things run (hence the PR into scmcallib). They run, but using the fixed time interval means you move off FaIR's native annual grid onto an evenly spaced one which seems to be having subtle effects on the results annoyingly. I think the solution is to make our timeseries stuff able to handle non-evenly spaced grids too.